### PR TITLE
Updated labels for parent or guardian

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/13016)

This PR is to address some 508 accessibility feedback we got on the 686 form. In the form there is a workflow where the user can choose to remove a stepchild from their benefits and in that there are a set of questions that ask the parent or guardian's first, middle, and last name. The accessibility team felt that it was not clear enough that we were asking for the parent or guardian's name and not the stepchild, particularly in the review screen, so this PR updates the labels for those questions to make the distinction explicit.

## Testing done
All existing unit tests still pass

## Screenshots

Before -

Screen Shot 2020-08-28 at 10 21 58 AM

After -

Screen Shot 2021-01-21 at 10 29 44 AM

## Acceptance criteria
- [x] As a user, I want to be sure the data I am submitting is correct
- [x] As a user, I want to understand what each data point represents, without seeing duplicate labels close to one another

### CODE REVIEW

File: src/applications/disability-benefits/686c-674/config/chapters/stepchild-no-longer-part-of-household/stepchild-information/stepchild-information.js [hunk 1]
```diff
@@ -47,18 +47,18 @@ export const uiSchema = {
...
+          'ui:title': 'First Name of parent or guardian',
           'ui:required': formData =>
             isChapterFieldRequired(
               formData,
               TASK_KEYS.reportStepchildNotInHousehold,
             ),
         },
         middle: {
-          'ui:title': 'Middle Name',
+          'ui:title': 'Middle Name of parent or guardian',
```
Comment (micahchiang) [inline src/applications/disability-benefits/686c-674/config/chapters/stepchild-no-longer-part-of-household/stepchild-information/stepchild-information.js:58]: Just a heads up, you can add a property called `hideEmptyValueInReview` to `ui:options` to hide this field on the review page - https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vre/definitions/profileAddress.js#L125

It's something that has come up in a11y audits of our other forms, may be worth while to do it here as well.

**Review by micahchiang — COMMENTED** [review]
In the old screenshot they've crossed out the military base description, that can also be handled with `hideEmptyValueInReview` 👍

File: src/applications/disability-benefits/686c-674/config/chapters/stepchild-no-longer-part-of-household/stepchild-information/stepchild-information.js (hunk 1 above)
Comment (cohnjesse) [reply src/applications/disability-benefits/686c-674/config/chapters/stepchild-no-longer-part-of-household/stepchild-information/stepchild-information.js:58]: @micahchiang That screenshot was used for two different tickets so the military thing isn't needed for this one.

File: src/applications/disability-benefits/686c-674/config/chapters/stepchild-no-longer-part-of-household/stepchild-information/stepchild-information.js [hunk 2]
```diff
@@ -47,18 +47,18 @@ export const uiSchema = {
       whoDoesTheStepchildLiveWith: {
         'ui:title': 'Who does this stepchild live with?',
         first: {
-          'ui:title': 'First Name',
+          'ui:title': 'First Name of parent or guardian',
```
Comment (micahchiang) [inline src/applications/disability-benefits/686c-674/config/chapters/stepchild-no-longer-part-of-household/stepchild-information/stepchild-information.js:50]: I think `Name` should be lowercased to `name` to maintain consistency with the other workflows.

**Review by micahchiang — APPROVED** [review]
lgtm.

File: src/applications/disability-benefits/686c-674/config/chapters/stepchild-no-longer-part-of-household/stepchild-information/stepchild-information.js (hunk 2 above)
Comment (cohnjesse) [reply src/applications/disability-benefits/686c-674/config/chapters/stepchild-no-longer-part-of-household/stepchild-information/stepchild-information.js]: true, updated

**Pull request merged**